k6/tests/heapMemory.js

```diff
@@ -37,15 +45,32 @@ export default function () {
     },
   });
 
-  // create payment
-  const payment = paymentsPage.createPayment(3);
-  check(payment, {
-    'Payment succesfull status was 202': (r) => r.status == 202,
-    'Payment has succesfully been initiated in less then 40s': (r) => {
-      if (r.timings.duration >= 40000) {
-        console.log(`Payment time was ${r.timings.duration}ms`);
+  // Do the payment
```

Comment:
I am not sure now that I read this again why these changes were made. Was not the purpose of this file to see how long it takes for a payment to start and than if a running payment affects the performance of other action you take in the platform? Or was it something else?

Should we rename this file from heapMemory.js to perfromanceDuringPayment.js or initiatingPaymentAndPerformanceDuringPayment or something like that?

Reply:
It was based on an issue we had couple of months ago and I think the work name was heap memory issue. Let's change it to:

perfromanceDuringPayment.js